pcsx2/R5900OpcodeImpl.cpp

```diff
@@ -882,10 +883,17 @@ void SYSCALL()
 
 	BIOS_LOG("Bios call: %s (%x)", R5900::bios[call], call);
 
-	switch (call) {
-		case 2: {
-					const char* inter = (cpuRegs.GPR.n.a0.UL[0] & 1) ? "Interlaced" : "Progressive";
-					const char* field = (cpuRegs.GPR.n.a2.UL[0] & 1) ? "FRAME" : "FIELD";
+
+	switch (static_cast<Syscall>(call)) {
+		case Syscall::SetGsCrt:
+		{
+					//Function "SetGsCrt(Interlace, Mode, Field)"
+					//Useful for fetching information of interlace/video/field display parameters of the Graphics Synthesizer
+
+					bool IsInterlaced = gsIsInterlaced = cpuRegs.GPR.n.a0.UL[0] & 1;
```

Comment:
`IsInterlaced` is useless now? You could use gsIsInterface directly, couldn't you?